README.md

```diff
@@ -20,6 +20,20 @@ Also `cache` should print `Getting from cache` when returns stored value and
 
 Example:
 ```python
+
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    def wrapper(*args, **kwargs): -> list
+        already_run = []
+        for i in args:
+            if
+            
+    
+
+
+
```

Comment:
Please don’t fix readme file

Reply:
yes, mistakenly was edited... how to revert it?

Reply:
you could copy the actual readme from mate academy repository and commit/push it